sample/build.gradle

```diff
@@ -27,6 +27,11 @@ android {
       proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
     }
   }
+
+  lintOptions {
+    abortOnError false
```

Comment:
Doing this is not a good idea, Travis won't catch potentially breaking bugs if Lint does not fail the build on an error.

Reply:
travis returned some false positive errors on menu_cab.xml. However I will revert this

Reply:
These are the errors:
`The first 3 errors (out of 5) were:
  /home/travis/build/afollestad/material-cab/sample/src/main/res/menu/menu_cab.xml:9: Error: Should use android:showAsAction when not using the appcompat library [AppCompatResource]
        app:showAsAction="ifRoom"/>
        ~~~~~~~~~~~~~~~~~~~~~~~~~
  /home/travis/build/afollestad/material-cab/sample/src/main/res/menu/menu_cab.xml:15: Error: Should use android:showAsAction when not using the appcompat library [AppCompatResource]
        app:showAsAction="ifRoom"/>
        ~~~~~~~~~~~~~~~~~~~~~~~~~
  /home/travis/build/afollestad/material-cab/sample/src/main/res/menu/menu_cab.xml:21: Error: Should use android:showAsAction when not using the appcompat library [AppCompatResource]
        app:showAsAction="ifRoom"/>
        ~~~~~~~~~~~~~~~~~~~~~~~~~`